CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+## 2.0.0
```

Comment:
Does it make sense to bump this to a major version?

Reply:
yes, due to vertx 4, a lot of modules become backward-incompatible. Hence it is a good idea to bump the major version as an indication for such.